pkg/errors/wrap_test.go

```diff
@@ -0,0 +1,56 @@
+package errors
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIs(t *testing.T) {
+
+	tests := []struct {
+		name        string
+		originalErr error
```

Comment:
Instead of calling this as originalErr and isErr, can we rename these as actualErr and expectedErr?

Reply:
left it as originalErr since that error is wrapped.  Both are actual errors.  

switch isErr to expErr